clients/unshaded/src/main/java/tachyon/client/file/options/MkdirOptions.java

```diff
@@ -44,6 +50,15 @@ public Builder setRecursive(boolean recursive) {
     }
 
     /**
+     * @param underStorageType the under storage type to use
+     * @return the builder
+     */
+    public Builder setUnderStorageType(UnderStorageType underStorageType) {
+      mUnderStorageType = underStorageType;
+      return this;
+    }
```

Comment:
Okay that makes sense, I originally thought there were only two outcomes we would consider.